app/templates/crate/settings.hbs

```diff
@@ -2,17 +2,20 @@
 
 <CrateHeader @crate={{this.crate}} />
 
-<h2>Add Owner</h2>
-
-<form local-class="email-form" {{on "submit" (prevent-default (perform this.addOwnerTask))}}>
-  <label local-class="email-input-label" for='new-owner-username'>
-    Username
-  </label>
-  <Input @type="text" id="new-owner-username" @value={{this.username}} placeholder="Username" local-class="email-input" name="username" />
-  <button type="submit" disabled={{not this.username}} class="button button--small" data-test-save-button>Save</button>
-</form>
+<div local-class="owners-header">
+  <h2>Owners</h2>
+  <button type="button" class="button button--small" data-test-add-owner-button {{on "click" this.showAddOwnerForm}}>Add Owner</button>
+</div>
 
-<h2>Owners</h2>
+{{#if this.addOwnerVisible}}
+  <form local-class="email-form" {{on "submit" (prevent-default (perform this.addOwnerTask))}}>
+    <label local-class="email-input-label" for='new-owner-username'>
+      Username
+    </label>
+    <Input @type="text" id="new-owner-username" @value={{this.username}} placeholder="Username" local-class="email-input" name="username" />
+    <button type="submit" disabled={{not this.username}} class="button button--small" data-test-save-button>Save</button>
```

Comment:
I tend to like specific verbs on forms, so maybe:

```suggestion
    <button type="submit" disabled={{not this.username}} class="button button--small" data-test-save-button>Add Owner</button>
```

Reply:
hahaha, I have a commit here already for a follow-up PR that pretty much does that 😂